### PR TITLE
HODS-377 | Changed documentation and areas in which code differs from…

### PR DESCRIPTION
**Pull request by matthewharris1**
… docs

### CODE REVIEW

File: resources/public/api/conf/1.0/examples/sautr-details-response-example.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "taxPayerName": "Partnership Name",
   "address": {
     "addressLine1": "123 Long Road",
-    "postcode": "AB12 3CD"
+    "postcode": "t"
```
[the remark this answers is not in the document]
Comment (parkeradam) [reply resources/public/api/conf/1.0/examples/sautr-details-response-example.json:7]:
```suggestion
    "postcode": "AB12 3CD"
```

**Review by parkeradam — CHANGES_REQUESTED** [review]
line 13 on spreadsheet is not complete. 
Line 20 on the spreadsheet is not complete.
Line 24 of the spreadsheet is not complete.
Line 26 of spreadsheet in not complete.

It also a bit hard to figure out if the actual responses do, I'll run this and hit endpoints to check response titles once the documentation has been fixed.

**Review by parkeradam — APPROVED** [review]

File: resources/public/api/conf/1.0/examples/corporation-tax-matching-response-example.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "_links": {
     "getCorporationTaxMatch": {
       "href": "/organisations/matching/corporation-tax/57072660-1df9-4aeb-b4ea-cd2d7f96e430",
-      "title": "Get the organisation's details"
+      "title": "Get links to Corporation Tax and number of employee details for a matched organisation"
```
Comment (Luka-Abey) [inline resources/public/api/conf/1.0/examples/corporation-tax-matching-response-example.json:5]:
```suggestion
      "title": "Get links to Corporation Tax and number of employees details for a matched organisation"
```

File: resources/public/api/conf/1.0/schemas/corporation-tax-matching-response.json [hunk 3]
```diff
@@ -29,13 +29,13 @@
             },
             "title": {
               "type": "string",
-              "description": "Get the organisation's details"
+              "description": "Get links to Corporation Tax and number of employee details for a matched organisation"
```
Comment (Luka-Abey) [inline resources/public/api/conf/1.0/schemas/corporation-tax-matching-response.json:32]:
```suggestion
              "description": "Get links to Corporation Tax and number of employees details for a matched organisation"
```

File: test/component/uk/gov/hmrc/organisationsmatchingapi/controllers/MatchingControllerSpec.scala [hunk 4]
```diff
@@ -93,7 +93,7 @@ class MatchingControllerSpec extends BaseSpec  {
            |  "_links" : {
            |    "getCorporationTaxMatch" : {
            |      "href" : "/organisations/matching/corporation-tax/$matchId",
-           |      "title" : "Get the organisation's details"
+           |      "title" : "Get links to Corporation Tax and number of employee details for a matched organisation"
```
Comment (Luka-Abey) [inline test/component/uk/gov/hmrc/organisationsmatchingapi/controllers/MatchingControllerSpec.scala:96]:
```suggestion
           |      "title" : "Get links to Corporation Tax and number of employees details for a matched organisation"
```

**Review by Luka-Abey — APPROVED** [review]

**Pull request merged**